.drone/drone.yml

```diff
@@ -1,30 +1,43 @@
 ---
 {
-   "kind": "pipeline",
-   "name": "validate-pr",
-   "steps": [
-      {
-         "commands": [
-            "make mod-check"
-         ],
-         "image": "golang:1.16.7",
-         "name": "make-mod-check"
-      },
-      {
-         "commands": [
-            "make lint"
-         ],
-         "image": "golang:1.16.7",
-         "name": "make-lint"
-      },
-      {
-         "commands": [
-            "make test"
-         ],
-         "image": "golang:1.16.7",
-         "name": "make-test"
-      }
-   ]
+  "kind": "pipeline",
+  "name": "validate-pr",
+  "steps": [
+    {
+      "commands": [
+          "make mod-check"
+      ],
+      "image": "golang:1.16.7",
+      "name": "make-mod-check"
+    },
+    {
+      "commands": [
+          "make lint"
+      ],
+      "image": "golang:1.16.7",
+      "name": "make-lint"
+    },
+    {
+      "commands": [
+          "make test"
+      ],
+      "image": "golang:1.16.7",
+      "name": "make-test"
+    },
+    {
+      "commands": [
+          "apt-get update && apt-get -y install unzip",
+          "make .tools/bin/protoc",
+          "go mod vendor",
+          "go get github.com/gogo/protobuf/protoc-gen-gogoslick@v1.3.0",
+          "go get github.com/golang/protobuf/protoc-gen-go@v1.3.1",
+          "go get github.com/gogo/protobuf/gogoproto@v1.3.0",
```

Comment:
Not a big fan of doing it here, cause then `make protos` wouldn't work out of the box on the CLI. For example, look what happens if I run the following:

```
$ make .tools/bin/protoc
$ make protos

.tools/protoc/bin/protoc -I /Users/marco/workspace:./vendor/github.com/gogo/protobuf:./vendor:./kv/memberlist --gogoslick_out=plugins=grpc,Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,:./kv/memberlist ./kv/memberlist/kv.proto
protoc-gen-gogoslick: program not found or is not executable
--gogoslick_out: protoc-gen-gogoslick: Plugin failed with status code 1.
make: *** [kv/memberlist/kv.pb.go] Error 1
```

What if we move these as a pre-requiste target for `protos`?